css/custom.css

```diff
@@ -51,10 +51,11 @@
   margin-bottom: 50px !important;
 }
 .newsletter-name{
-  padding-right: 0px;
+  padding-right: 5px !important;
```

Comment:
Why used `!important`
It's not healthy.

Reply:
Okay @0x48piraj , I should try to solve it otherway around.

Reply:
@0x48piraj No other option.
That's one only.

See I have uploaded the different images of different screen size

![f1](https://user-images.githubusercontent.com/22450186/34914152-a3ea848c-f932-11e7-903b-ed854804b27b.png)
![f2](https://user-images.githubusercontent.com/22450186/34914153-a55653fa-f932-11e7-8582-d41ebe9351c5.png)

It's working.

Reply:
@0x48piraj kindly review it.